creating-virtual-machines/interfaces-and-networks.adoc

```diff
@@ -238,15 +238,68 @@ spec:
     pod: {}
 ----
 
+______________________________________________________________________________
+**Note:** If a specific MAC address is configured in the virtual machine
+interface spec, it's passed to underlying CNI plugin that should configure
+backend to allow for this particular address. While some plugins are known to
+work without special support for custom MAC addresses, others do (for example,
+OpenShift SDN needs to configure OVS flows to allow traffic meant for the
+chosen MAC address).
+______________________________________________________________________________
+
+______________________________________________________________________________
+**Note:** If a plugin doesn't support `mac` attribute defined by Kubernetes
+Network Plumbing Working Group, one can use `tuning` plugin that is one of
+reference plugins maintained by the group. This can be used as follows:
+
+[source,yaml]
+----
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata:
+  name: ptp-mac
+spec:
+  config: '{
+      "cniVersion": "0.3.1",
+      "name": "ptp-mac",
+      "plugins": [
+        {
+          "type": "ptp",
+          "ipam": {
+            "type": "host-local",
+            "subnet": "10.1.1.0/24"
```

Comment:
Since it's all about MAC maybe add mac address to the example yaml?

Reply:
@myakove which example should I add it to? From kubevirt perspective, I would need to add a separate example with macAddress attribute. The MAC address is not part of network CRD. Do you want me to update the VM spec defined in line 223+?